openedx/features/genplus_features/genplus/models.py

```diff
@@ -45,39 +42,56 @@ def __str__(self):
         return self.name
 
 
+class TempUser(TimeStampedModel):
```

Comment:
what is the use case of this model?

Reply:
We are fetching users/students AOT from rmunify this model is to store the users which are not yet registered to the Gen+ site